.github/dependabot.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directories:
+      - "/"
+      - "/examples"
+      - "/plugins/**/*"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10
+    groups:
+      golang-x:
+        patterns:
+          - "golang.org/x/*"
+      google-golang:
+        patterns:
+          - "google.golang.org/*"
+      containerd:
+        patterns:
+          - "github.com/containerd/*"
+      opencontainers:
+        patterns:
+          - "github.com/opencontainers/*"
```

Comment:
Q: Do we need the opencontainers dependency added, since most of the dependencies are related to runtime-spec?

Reply:
> Do we need the opencontainers dependency added

This only groups dependencies matching the format into a single PR.

From my local test, using a group explicitly makes dependabot more likely update the dependencies in all `go.mod` in a single PR, so the CI can pass.

I'm not sure why, since dependabot should update a dep in all `go.mod` regardless if group or not. So I think we can use groups/patterns for now which doesn't hurt anyway.

> E.g., the 1st PR was created with `patterns`, which updates oci in all `go.mod`; the other 4 PRs update the same dep but in separate PRs.

<img width="1016" alt="image" src="https://github.com/user-attachments/assets/cc8d2466-9578-48da-bb45-1812298eab2f" />

Reply:
Ah. Got it. Thank you 👍